netjsonconfig/schema.py

```diff
@@ -469,6 +549,130 @@
+                    "title": "accounting port",
+                    "default": 1813,
+                    "propertyOrder": 8,
+                },
+            },
+        },
+        "encryption_wpa_enterprise_sta_base_settings": {
+            "properties": {
+                "eap_type": {
+                    "title": "EAP protocol",
+                    "type": "string",
+                    "enum": ["tls", "ttls"],
+                    "options": {"enum_titles": ["EAP-TLS", "EAP-TTLS"]},
+                    "propertyOrder": 4,
+                },
+                "identity": {"type": "string", "propertyOrder": 5},
+                "password": {"type": "string", "propertyOrder": 6},
+                "ca_cert": {
+                    "type": "string",
+                    "title": "CA certificate (path)",
+                    "propertyOrder": 7,
+                },
+                "client_cert": {
+                    "type": "string",
+                    "title": "client certificate (path)",
+                    "propertyOrder": 8,
+                },
+                "priv_key": {
+                    "type": "string",
+                    "title": "private key (path)",
+                    "propertyOrder": 9,
+                },
+                "priv_key_pwd": {
+                    "type": "string",
+                    "title": "private key password",
+                    "propertyOrder": 10,
+                },
+            },
+        },
+        "encryption_wpa3_enterprise_base_settings": {
+            "properties": {
+                "protocol": {
+                    "type": "string",
+                    "title": "encryption protocol",
+                    "enum": ["wpa3_enterprise"],
+                    "options": {"enum_titles": ["WPA3 Enterprise"]},
+                    "propertyOrder": 1,
+                }
+            }
+        },
+        "encryption_wpa3_2_enterprise_base_settings": {
+            "properties": {
+                "protocol": {
+                    "type": "string",
+                    "title": "encryption protocol",
+                    "enum": ["wpa2_enterprise_mixed"],
+                    "options": {"enum_titles": ["WPA3/WPA2 Enterprise Mixed Mode"]},
+                    "propertyOrder": 1,
+                }
+            }
+        },
+        "encryption_wpa3_enterprise_ap": {
+            "title": "WPA3 only Enterprise (access point)",
+            "allOf": [
+                {"$ref": "#/definitions/encryption_base_settings"},
+                {"$ref": "#/definitions/encryption_cipher_property"},
+                {"$ref": "#/definitions/encryption_mfp_property_required"},
+                {"$ref": "#/definitions/encryption_wpa3_enterprise_base_settings"},
+                {"$ref": "#/definitions/encryption_wpa_enterprise_ap_base_settings"},
+            ],
+        },
+        "encryption_wpa3_2_enterprise_ap": {
+            "title": "WPA3/WPA2 Enterprise (access point)",
```

Comment:
I think we can make it explicit here that it is mixed mode. 
```suggestion
            "title": "WPA3/WPA2 Enterprise (access point) Mixed Mode",
```